ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -96,6 +98,15 @@ public void applyComponentOrientation(ComponentOrientation o) {
     }
   };
 
+  class DecimalRenderer extends DefaultTableCellRenderer {
+    private final DecimalFormat myFormatter = new DecimalFormat("#0.00");
```

Comment:
The format of representing currency values is locale-specific. You want to use [NumberFormat::getCurrencyInstance](https://docs.oracle.com/javase/8/docs/api/java/text/NumberFormat.html#getCurrencyInstance-java.util.Locale-) and pass the currently selected locale. `GanttLanguage` is responsible for such things (and it stores the current locale)